core/examples/driver/main.cpp

```diff
@@ -75,12 +75,20 @@ int main(int /*argc*/, char** /*argv*/)
 {
```

Comment:
Trying to run the `usCoreExamplesDriver` executable on my Fedora 22 box crashes with the following output:

```
[sascha@bigeye-nb bin]$ ./usCoreExamplesDriver 
ServiceTracker<S,TTT>::Open: 1
terminate called after throwing an instance of 'std::runtime_error'
  what():  module.name is not defined in the bundle manifest.
Aborted (core dumped)
```
Does it run for you?